src/lambda_function.py

```diff
@@ -1,10 +1,10 @@
 from aws_xray_sdk.core import xray_recorder
 from aws_xray_sdk.core import patch
 import json
-import getCredibilityScore as cr
-import sentimentAnalysis as sa
-import getBiasScore as bs
-import spacyMatcher as sm
+import get_credibility_score as cr
```

Comment:
I'm not a massive fan of using these "nicknames" for things. What's wrong with just importing `get_credibility_score` and calling it by that name when you use it?